src/types.ts

```diff
@@ -74,3 +74,9 @@ export type ECPublicKey = string;
 export type EventListenerID = string;
 
 export type NetworkString = 'liquid' | 'testnet' | 'regtest';
+
+// return object from sendTransaction
+export interface SentTransaction {
+  txid: TransactionID;
+  hex: PsetBase64;
```

Comment:
not really in PSET base64 format, we output here the extracted raw hex transaction.

```suggestion
  hex: RawHex;
```

Reply:
Fixed by c94a188